src/create_input_qe.py

```diff
@@ -227,7 +233,7 @@ def write_qe_input(ex_st, label, mol, params,occ,occ_alp,occ_bet):
         
     g.close()
 
-def write_qe_input_first(filename,occ,occ_alp,occ_bet,nspin):
+def write_qe_input_first(filename,occ,occ_alp,occ_bet,nspin,params,restart_flag):
```

Comment:
likewise, please document all the input variables and the purpose of the function

Reply:
Done!